build/Build.cs

```diff
@@ -815,7 +815,11 @@ private async Task<string> SendSwaggerSchemaToValidator(HttpClient httpClient, s
 
             var processEnd = ProcessTasks.StartProcess(dotNetPath, endCmd, customLogger: SonarLogger, logInvocation: false)
                 .AssertWaitForExit().AssertZeroExitCode();
-            processEnd.Output.EnsureOnlyStd();
+            var errors = processEnd.Output.Where(o => !o.Text.Contains(@"The 'files' list in config file 'tsconfig.json' is empty") && o.Type == OutputType.Err);
```

Comment:
Prevent double enumeration.
```suggestion
            var errors = processEnd.Output.Where(o => !o.Text.Contains(@"The 'files' list in config file 'tsconfig.json' is empty") && o.Type == OutputType.Err).ToList();
```